.github/workflows/build.yml

```diff
@@ -16,10 +16,9 @@ jobs:
         uses: actions/checkout@v3
 
       - name: Install stable toolchain
-        uses: actions-rs/toolchain@v1
+        uses: helix-editor/rust-toolchain@v1
```

Comment:
Why do we need our own fork?

Reply:
We merged in https://github.com/actions-rs/toolchain/pull/209 - without it we would need to use the deprecated `rust-toolchain` file rather than `rust-toolchain.toml`.

Looks like upstream might be unmaintained at the moment as well but we'll leave https://github.com/helix-editor/helix/issues/2528 open to switch back if 209 gets merged upstream.